web-ui/view/api/git_forge.ml

```diff
@@ -0,0 +1,50 @@
+module type Api = sig
+  module Client = Ocaml_ci_api.Client
+  module Run_time = Ocaml_ci_client_lib.Run_time
+
+  val show_step :
+    step_info:Client.job_info option ->
+    run_time:Run_time.run_time_info option ->
+    can_rebuild:bool ->
+    Dream.response Lwt.t
+
+  val list_steps :
+    org:string ->
+    repo:string ->
+    hash:string ->
+    jobs:Client.job_info list ->
+    build_status:Client.State.t ->
+    Dream.response Lwt.t
+end
+
+module type M_Git_forge = sig
+  val prefix : string
+end
```

Comment:
Following an IRL chat this module is very thin at the moment but has the potential to change when we properly implement the GitLab views. If the module does not change and is just a wrapper on a string, maybe we back away from this approach and just used shared code between the different git-forges with the prefix injected into this shared code.